models.py

```diff
@@ -22,10 +17,24 @@ def __init__(self, nom, prenom, date_de_naissance, sexe, classement):
             self.prenom = prenom
             self.date_de_naissance = date_de_naissance
             self.sexe = sexe
-            self.classmeent = classement
+            self.classement = classement
+
+    def __str__(self):
+        return f"{self.nom},{self.prenom},{self.date_de_naissance},{self.sexe},{self.classement}"
+
+
+
+
+class Tours:
+    def __init__(self, add_players, enter_results ):
+        self.add_players = add_players
+        self.enter_results = enter_results
+
 
 
 
 
+class Matchs:
```

Comment:
I think you need to go back to the exercise to complete this.

What is needed to create a match? players? a tournament? a tour?